permits/views.py

```diff
@@ -178,27 +190,55 @@ def permit_request_appendices(request, permit_request_id):
 
 @login_required
 def permit_request_actors(request, permit_request_id):
+
     permit_request = services.get_permit_request_for_user_or_404(request.user, permit_request_id)
-    GenericActorFormSet = modelformset_factory(Actor, form=ActorForm, extra=0)
+
+    unique_initial_actors = models.PermitActorType.objects.filter(
+        works_type__in = services.get_permit_request_works_types(permit_request)
+    ).exclude(type__in=models.PermitRequestActor.objects.filter(permit_request=permit_request).values_list('actor_type',flat=True)).distinct('type')
+
+    actor_initial_forms = []
```

Comment:
`actor_initial_forms = [{'actor_type': actor_type.type} for actor_type in unique_initial_actors]`